src/test/java/com/google/api/codegen/protoannotations/testdata/book_from_anywhere.proto

```diff
@@ -0,0 +1,32 @@
+// Intended to test resource name generation for a message from a file without
+// any RPCs
+
+syntax = "proto3";
+
+package google.example.library.v1;
+
+import "google/api/annotations.proto";
+
+option go_package = "google.golang.org/genproto/googleapis/example/library/v1;library";
+option java_multiple_files = true;
+option java_outer_classname = "BookFromAnywhereProto";
+option java_package = "com.google.example.library.v1";
+
+// A single book in the archives or library.
+message BookFromAnywhere {
+  // The resource name of the book.
+  // Book names have the form `bookShelves/{shelf_id}/books/{book_id}` or
+  // `archives/{archive_id}/books/{book_id}`.
+  string name = 1 [
+    (google.api.resource_type) = "google.example.library.v1.Book"
```

Comment:
This entire file was copied so that we could have a version of book_from_anywhere.proto with this line that specifies a resource type.